tests/unit/polars_session.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dataclasses
+from typing import Mapping, Optional, Union
+import weakref
+
+import polars
```

Comment:
Should use `polars = pytest.importorskip("polars")`

Or maybe not necessary since this isn't a `_test.py` file.

Reply:
seems to be running fine for now as importing only in fixtures used for polars tests